src/activity_handlers.cpp

```diff
@@ -3603,6 +3621,17 @@ void activity_handlers::hacksaw_finish( player_activity *act, player *p )
     } else if( ter == t_door_bar_c || ter == t_door_bar_locked ) {
         here.ter_set( pos, t_mdoor_frame );
         here.spawn_item( p->pos(), itype_pipe, 12 );
+    } else if( ter == t_metal_grate_window || ter == t_metal_grate_window_with_curtain ||
+               ter == t_metal_grate_window_with_curtain_open ) {
+        here.ter_set( pos, t_window_reinforced );
+        here.spawn_item( p->pos(), itype_rebar, rng( 1, 12 ) );
```

Comment:
Just an FYI, it looks like the metal_grate_window uses pipes and not rebar in it's construction.  It probably *should* be using rebar, but for the moment it's pipes

Reply:
Yeah, I noticed that too. Just an oversight from my side. Will fix this soon.